src/_cffi_src/openssl/ssl.py

```diff
@@ -434,6 +434,10 @@
 long SSL_CTX_sess_misses(SSL_CTX *);
 long SSL_CTX_sess_timeouts(SSL_CTX *);
 long SSL_CTX_sess_cache_full(SSL_CTX *);
+
+/* DTLS support */
+long DTLSv1_get_timeout_wrapped(SSL *, time_t *, long int *);
```

Comment:
Style nit: Let's just declare the last arg as `long` (which is more consistent with our naming conventions).